components/movementsensor/imuvectornav/imu.go

```diff
@@ -24,12 +24,15 @@ import (
 
 var model = resource.DefaultModelFamily.WithModel("imu-vectornav")
 
+// DefaultBaudRate is the value we assign if there is no value specified in the attributes.
+const DefaultBaudRate = 9600
```

Comment:
```suggestion
const defaultBaudRate = 9600
```
no need for this to be exported.